DataFormats/GeometryVector/src/classes_def.xml

```diff
@@ -14,11 +14,11 @@
   <class name="VectorTag" ClassVersion="10">
    <version ClassVersion="10" checksum="898711"/>
   </class>
-  <class name="Geom::Phi<double, Geom::PhiRange::MinusPiToPi>" ClassVersion="10">
-   <version ClassVersion="10" checksum="2898804398"/>
+  <class name="Geom::Phi<double, Geom::MinusPiToPi>" ClassVersion="11">
```

Comment:
There is no reason to increment the class version: please stay with "10"

Reply:
Before I updated this file, the compiler gave an error and specified a new checksum and version number. I was only following the instructions from the compiler message. I can try decrementing the version number and see if the compiler complains.

Reply:
You have technically cretated a new class because the template type is different. In that case you should use ClassVersion=3 (we used 10 originally because we needed to be safe reading ROOT files generated before we started adding ClassVersions to the classes_def.xml)

Reply:
@Dr15Jones: With the specification of ClassVersion=3, will the code still be able to correctly read old persisted data with the original version of the `Phi` class?

Reply:
The answer to my own question is Yes, old persisted data can be read by the new code.